docs/standard/microservices-architecture/multi-container-microservice-net-applications/implementing-api-gateways-with-ocelot.md

```diff
@@ -0,0 +1,579 @@
+---
+title: Implementing API Gateways with Ocelot
+description: .NET Microservices Architecture for Containerized .NET Applications | In this section you can learn how to implement API Gateways with Ocelot, which is an open sourced and lightweight API Gateway based on .NET Core. You will also learn how to use Ocelot in a container-based environment.
```

Comment:
This is too long. Book title should not be here. 

Suggestion: Learn how to implement API Gateways with Ocelot and how to use Ocelot in a container-based environment.

Reply:
OK, changed the title and tags for no C# (like YAML) are now as simply ```
However, I still want to keep "ing" for this specific section as it is not imperative to use Ocelot. Ocelot is just one possible option.
I'll change the filename, as you suggest.  :)